lib/events/auditlog.go

```diff
@@ -1235,3 +1309,10 @@ func (a *closedLogger) WaitForDelivery(context.Context) error {
 func (a *closedLogger) Close() error {
 	return trace.NotImplemented(loggerClosedMessage)
 }
+
+func (a *closedLogger) StreamSessionEvents(_ctx context.Context, sessionID session.ID, startIndex int) (chan apievents.AuditEvent, chan error) {
+	e, c := make(chan error, 1), make(chan apievents.AuditEvent)
+	e <- trace.NotImplemented(loggerClosedMessage)
+	close(c)
+	return c, e
```

Comment:
For this and all the other unimplemented `StreamSessionEvents`: this may not matter if they are never called, but if a caller does a `select` on `e` and `c` I don't think it's deterministic whether they will see the error or just the closed events channel.

Solutions I see would be:
1. Don't close `c`. Then we're requiring the caller to select on both but maybe that's okay
2. Don't buffer `e`, and spawn a goroutine to send the error then close `c`

Reply:
I went with solution 2 and applied it to all unimplemented variants.

Reply:
I see you fixed it in a bunch of places but ironically not here I think